service-deployment/clusterconfig-example.yaml

```diff
@@ -26,6 +26,9 @@ clusterinfo:
   # PAI data root path
   paiPath: "/var/pai"
 
+  # Choose proper nvidia driver version from this url http://www.nvidia.com/object/linux-amd64-display-archive.html
+  nvidia_drivers_version: "384.111"
```

Comment:
Next time, should we update the deployment in PaiDeployment git instead?

Reply:
Yes, I will update it.